Description: Upstream changes introduced in version 3.0.5-4
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 openhrp3 (3.0.5-4) lucid; urgency=low
 .
   * Add java3d to dependencies.
   * Fix desktop file.
   * Fix config.sh file before installing it.
 .
 The person named in the Author field signed this changelog entry.
Author: Thomas Moulard <thomas.moulard@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ openhrp3-3.0.5/Make.vars
@@ -0,0 +1,141 @@
+# -*- Makefile -*-
+#
+
+# Select one of the DynamicsSimulators
+# ut   : the server developed by the University of Tokyo
+# aist : the server developed by the AIST
+DYNAMICS_ENGINE = aist
+
+# ORB installed directory
+# Only use if not installed in the default include path such as 
+# /usr/include or /usr/local/include
+OB_DIR =
+
+#
+# Java Development Kit installed directory.
+# Only use If not installed in the default JDK's PATH such as /usr/bin
+# update-java-alternatives
+JDK_DIR = 
+
+#
+# Boost header directory
+# Only use if not installed in the default include path such as
+# /usr/include or /usr/local/include
+BOOST_INC_DIR = 
+
+# Boost binary library directory
+# Only use if not installed in the default library path such as /usr/lib
+BOOST_LIB_DIR = 
+
+# compiler identifier of the boost binaries
+# Ubuntu 7.04
+# BOOST_LIB_SFX = -gcc-mt-1_33_1
+# Ubuntu 7.10
+BOOST_LIB_SFX = -mt
+
+#
+# tvmet installed directory
+# Only use if not installed in the default include path such as 
+# /usr/include or /usr/local/include
+TVMET_DIR = 
+
+#
+# Naming Service location
+NS_HOST = localhost
+NS_PORT = 2809
+
+
+# You can specify variants to LAPACK using the following LAPACK_VARIANT
+# variable. One of the following values must be set:
+#  LAPACK  - The original LAPACK written in Fortran
+#  CLAPACK - CLAPACK 
+#  ATLAS
+#  MKL     - Intel's Math Kernel Library
+LAPACK_VARIANT = ATLAS
+
+# Lapack header file directory
+# Only use if not installed in the default include path such as 
+# /usr/include or /usr/local/include
+LAPACK_INC_DIR = 
+
+# Lapack lib file directory
+# Only use if not installed in the default library path such as LAPACK_VARIANT_LIB_DIR
+LAPACK_LIB_DIR = 
+
+# Lapack additional lib file directory for LAPACK variants such as ATLAS
+LAPACK_VARIANT_LIB_DIR = /usr/lib/atlas
+
+# Set the "PLAT" value defined in "make.inc" of the CLAPACK compilation directory
+# when you use CLAPACK made from the source package.
+CLAPACK_SUFFIX =  _LINUX
+
+
+# OpenRTM installed directory
+# Only use if not installed in the default include path such as
+# /usr/include or /usr/local/include
+OPENRTM_DIR =
+
+#
+# jython installed directory
+JYTHON_HOME = /usr/share/jython
+JYTHON_JAR = /usr/share/java/jython.jar
+JYTHON_CACHE = $(HOME)/.jython-cache
+
+# java ext directory
+JAVAEXT_DIR = /usr/share/java
+
+# JMF installed directory
+# Only use if not use in the default include path such as
+# $(OPENHRPHOME_DIR)/client/gui/JMF2.1.1e/unix
+JMFHOME = 
+
+# Java3D installed directory.
+# Only use if not installed in the default include path such as
+# $(OPENHRPHOME_DIR)/client/gui/Java3D/1.4.1/unix directory
+J3DHOME = 
+
+#
+# Intel Compiler Options
+#
+#
+# set following to 1 to compile with the Intel Compiler
+
+#USE_INTEL_COMPILER = 1
+
+#
+# use the following to set Intel Compiler version
+# To compile Corba Generated C/C++ files you must use
+# version 10.xx and up.
+
+# INTEL_COMPILER_VERSION = 10.1.011
+
+#
+# use the following to set the Intel MKL version
+# INTEL_MKL_VERSION = 10.0.1.014
+
+
+# Intel Compiler: Profile Guided Optimization
+# Compile with INTEL_PROFILE_GENERATE=1.
+# Execute binary and then compile again with
+# INTEL_PROFILE_GENERATE=0
+#
+#INTEL_PROFILE_GENERATE = 1
+
+
+# Arch i386/amd64 for Java and Intel Compiler and MKL
+M_ARCHITECTURE = i386
+
+# You can set additional C/C++ Compiler options for a specific CPU 
+# CXX_CPU_FLAGS = -mtune=pentium-m -msse3 -mfpmath=sse
+
+
+#
+# Set use of OpenMP directives
+#
+#OPENMP = 1
+
+# Set one of the following values to DEBUG_VER
+#  0 - Release mode with optimization
+#  1 - Debug mode with optimization
+#  2 - Debug mode without optimization
+DEBUG_VER = 0
--- openhrp3-3.0.5.orig/bin/unix/Makefile
+++ openhrp3-3.0.5/bin/unix/Makefile
@@ -13,7 +13,7 @@ OpenHRPpy:$(TOP)Make.rules $(TOP)Make.va
 	chmod a+x OpenHRPpy
 
 config.sh:$(TOP)Make.rules $(TOP)Make.vars Makefile
-	/bin/echo 'export OPENHRPHOME=$(OPENHRPHOME_DIR)'> config.sh
+	/bin/echo 'export OPENHRPHOME=/opt/OpenHRP-3.0.5'> config.sh
 	/bin/echo >> config.sh
 	/bin/echo 'export DYNAMICS_ENGINE=$(DYNAMICS_ENGINE)'>> config.sh
 	/bin/echo 'export JYTHON_HOME=$(JYTHON_HOME)'>> config.sh
